Update cardano-mdp.json

setiamanlhc opened the pull request:
add KAYA Pool entry #116

BrianLee12 commented:
Hey setiamanlhc, thanks for posting your donation receipt to your website. I see that it is in your blog, which will be difficult to find when you get many posts in there. It would be better to have an easy to find link (say on your main page) to all your receipts.
Also, when you get a chance, please add your receipt to the MDP GitHub as well so we can easily keep a running total for everyone.
Thanks! 
Brian

setiamanlhc commented:
Hi Brian,

I have added donation.yml and also check in the receipts accordingly.
In addition I have added new page to consolidate all donation in my website
here: https://kayapool.com/donation/
Let me know what I need to do in order by PR to be approved.
I saw it is still hanging and I cannot do anything on that PR.

Regards,
Setiaman


On Mon, Sep 13, 2021 at 10:41 AM GAIA Stake Pool ***@***.***>
wrote:

> Hey setiamanlhc, thanks for posting your donation receipt to your website.
> I see that it is in your blog, which will be difficult to find when you get
> many posts in there. It would be better to have an easy to find link (say
> on your main page) to all your receipts.
> Also, when you get a chance, please add your receipt to the MDP GitHub as
> well so we can easily keep a running total for everyone.
> Thanks!
> Brian
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CardanoMDP/CardanoMDP-adapools-org-alliance/pull/127#issuecomment-917790244>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMEMUWI2B6FPGXD7RWYFI3UBVQFBANCNFSM5DNLJIRA>
> .
>
